### ISSUE TITLE
Dream Request

### ISSUE THREAD

**Issue by yzhlau** [Ixaruz/Luna-Archive-of-Dreams#75]
Hi, can you dump this dream request ? Thanks

DA-2557-1607-2056
DA-9258-5523-0303
DA-3328-3405-6427
DA-1793-1012-1187

**Comment by somegenie** [comment]
OK, done all 4. Sorry I'm not sure how to request to add them to the archive yet, would appreciate if anyone knows how.

**Comment by Ixaruz** [comment]
@somegenie you can try opening a pull request, if it isn't too much hastle for you. Otherwise I would add the dumps on the weekend myself lol.

**Comment by somegenie** [comment]
Could I get a step-by-step maybe? I'm sorry, I'm looking at the GitHub Docs for a guide and I'm still lost... I'm not sure how to create a pull request. I tried going to the "labels" menu and that takes me back to the issues. I did see an issue that was an archive addition, though, is that what I'm supposed to do? Again, sorry for being dumb and completely lost but I'd love to help if I can figure out how. I don't know if I'm supposed to do this but it lets me attach files to this comment so I guess I'll include one here? I hope that's not a wrong thing.
[[DA-1793-1012-1187].zip](https://github.com/Ixaruz/Luna-Archive-of-Dreams/files/7263881/DA-1793-1012-1187.zip)
